### DISCUSSION TITLE
wiki/pyqgis-sd/04_prev/status

### DISCUSSION THREAD

**Discussion by giscus** [SX-Code/blog-comments#18] [Announcements]
# wiki/pyqgis-sd/04_prev/status

敬请期待

https://sx-code.github.io/wiki/pyqgis-sd/04_prev/status.html

**Comment by wiaini** [comment]
写的太好了，给你点赞，不过状态栏信息显示怎么没有内容呢

**Reply by SX-Code** [reply]
已补充，请注意查收:)
